app/src/main/java/com/nextcloud/talk/controllers/ConversationsListController.java

```diff
@@ -445,7 +445,9 @@ private void fetchData(boolean fromBottomSheet) {
 
         callItems = new ArrayList<>();
 
-        int apiVersion = ApiUtils.getConversationApiVersion(currentUser, new int[]{ApiUtils.APIv4, ApiUtils.APIv3, 1});
+        int apiVersion = ApiUtils.getConversationApiVersion(
+            currentUser,
+            new int[]{ApiUtils.APIv4, ApiUtils.APIv3, 2, 1});
```

Comment:
No, that's not it. v1 covers everything until including Nextcloud 21 and v4 covers 22